api/app/fixtures/inti_data.yaml

```diff
@@ -0,0 +1,1866 @@
+- model: auth.permission
+  pk: 199
+  fields:
+    name: Can delete segment configuration
+    content_type: 51
+    codename: delete_segmentconfiguration
+- model: auth.permission
+  pk: 200
+  fields:
+    name: Can view segment configuration
+    content_type: 51
+    codename: view_segmentconfiguration
+- model: auth.permission
+  pk: 201
+  fields:
+    name: Can add heap configuration
+    content_type: 52
+    codename: add_heapconfiguration
+- model: auth.permission
+  pk: 202
+  fields:
+    name: Can change heap configuration
+    content_type: 52
+    codename: change_heapconfiguration
+- model: auth.permission
+  pk: 203
+  fields:
+    name: Can delete heap configuration
+    content_type: 52
+    codename: delete_heapconfiguration
+- model: auth.permission
+  pk: 204
+  fields:
+    name: Can view heap configuration
+    content_type: 52
+    codename: view_heapconfiguration
+- model: auth.permission
+  pk: 205
+  fields:
+    name: Can add mixpanel configuration
+    content_type: 53
+    codename: add_mixpanelconfiguration
+- model: auth.permission
+  pk: 206
+  fields:
+    name: Can change mixpanel configuration
+    content_type: 53
+    codename: change_mixpanelconfiguration
+- model: auth.permission
+  pk: 207
+  fields:
+    name: Can delete mixpanel configuration
+    content_type: 53
+    codename: delete_mixpanelconfiguration
+- model: auth.permission
+  pk: 208
+  fields:
+    name: Can view mixpanel configuration
+    content_type: 53
+    codename: view_mixpanelconfiguration
+- model: auth.permission
+  pk: 209
+  fields:
+    name: Can add access attempt
+    content_type: 54
+    codename: add_accessattempt
+- model: auth.permission
+  pk: 210
+  fields:
+    name: Can change access attempt
+    content_type: 54
+    codename: change_accessattempt
+- model: auth.permission
+  pk: 211
+  fields:
+    name: Can delete access attempt
+    content_type: 54
+    codename: delete_accessattempt
+- model: auth.permission
+  pk: 212
+  fields:
+    name: Can view access attempt
+    content_type: 54
+    codename: view_accessattempt
+- model: auth.permission
+  pk: 213
+  fields:
+    name: Can add access log
+    content_type: 55
+    codename: add_accesslog
+- model: auth.permission
+  pk: 214
+  fields:
+    name: Can change access log
+    content_type: 55
+    codename: change_accesslog
+- model: auth.permission
+  pk: 215
+  fields:
+    name: Can delete access log
+    content_type: 55
+    codename: delete_accesslog
+- model: auth.permission
+  pk: 216
+  fields:
+    name: Can view access log
+    content_type: 55
+    codename: view_accesslog
+- model: users.ffadminuser
+  pk: 1
+  fields:
+    password: pbkdf2_sha256$150000$ZgVj2otuj0jc$DTiJ9444rkdUMdSr7YPKuBgPZFInh5wyb2TAH0V1Fsg= # test123
+    last_login: 2021-07-20 04:57:47.633424+00:00
+    is_superuser: true
+    is_staff: true
+    is_active: true
+    date_joined: 2021-07-20 04:53:29.305413+00:00
+    email: local@flagsmith.com
```

Comment:
I wonder if, instead of this, we could just have a script that essentially wraps this but takes the email and password as an input? 

Something like (forgive my bash - this is pretty much freehand): 

```
#!/bin/bash

emailAddress=$1
password=$2

sed -e 's/EMAIL_ADDRESS/$emailAddress' -e 's/PASSWORD/$password' api/app/fixtures/init_data.yml.template > api/app/fixtures/init_data.yml
python manage.py loaddata api/app/fixtures/init_data.yml
```

Note that we'd probably also then want to gitignore the init_data.yml